diff --git a/CMakeLists.txt b/CMakeLists.txt
index f7301a43..eb22c99e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,10 +15,10 @@
 set(CMAKE_LEGACY_CYGWIN_WIN32 0) # needed to silence warning on cygwin
 
 # require cmake 2.6.2 (but please use 2.8.x)
-cmake_minimum_required(VERSION 2.6.2 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 
 # the STXXL project
-project(stxxl)
+project(stxxl VERSION 1.4.1)
 
 # for additional cmake scripts
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/misc/cmake)
@@ -742,7 +742,7 @@ install(EXPORT stxxl-targets DESTINATION "${INSTALL_CMAKE_DIR}")
 ###############################################################################
 # build tests and tools
 
-add_subdirectory(tools)
-add_subdirectory(examples)
-add_subdirectory(tests)
-add_subdirectory(local)
+# add_subdirectory(tools)
+# add_subdirectory(examples)
+# add_subdirectory(tests)
+# add_subdirectory(local)
